\section{Introduction}

\begin{itemize}

\item What is the subject of the study? Describe the
economic/econometric problem.

-> Implementation of Cancer Gene Panels & Variant Calling Algorithms

\item What is the purpose of the study (working hypothesis)?

-> Check whether one of them is better than the others.
-> Study effect of FFPE

\item What do we already know about the subject (literature
review)? Use citations: { \cite{mutect:2013} shows that...
Alternative Forms of the Wald test are considered
\citep{Breusch&Schmidt:88}.}

\item What is the innovation of the study?

-> relatively rare technique in diagnostics, but has many advantages

\item Provide an overview of your results.


\end{itemize}

General Intro; maybe subsection about

\subsection{EGFR Signaling Cascade}

Describe pathway

Common mutations in this pathway

EGFR-targeted drugs

\subsection{Targeted Sequencing and Target Enrichment Methods}

Hybrid capture

Selective circularization

PCR amplification

\subsection{Illumina MiSeq Sequencing Chemistry}

Picture

\subsection{NGS Data Analysis}

GATK best practices

\subsection{Practical Implications in the Laboratory}

FFPE : more details

\subsection{Aims of the Thesis}
Targeted NGS is still not widely used in diagnostics laboratories. The SGMB of the LNS
is planning to build expertise with the aim to adopt NGS routinely in the laboratory,
mainly in the context of diagnosis and therapy of cancer patients in Luxembourg.

The aim of this thesis project was to test commercially available cancer gene panels,
e.g. Illumina Trusight Tumor 15 and Agilent Haloplex HS ClearSeq Cancer, for their
potential use in the routine workflow of the laboratory. Several samples of cancer
patients were prepared with both kits and were sequenced on the Illumina MiSeq device.
Both kits vary in their sequencing library preparation principles: Illumina's tst15
uses the multiplex PCR approach while Agilent's Haloplex Enrichment System uses
enzymatic DNA restriction followed by probe capture.

NGS data were analyzed with the respective recommended pipelines and a custom in-house
pipeline.

Finally, several freely available variant calling algorithms were tested for their
potential implementation in the custom in-house variant discovery bioinformatic
pipeline.
